### ISSUE TITLE
dual carriageway as a separate components

### ISSUE THREAD

**Issue by antblant** [ppawel/osmonitor#39]
http://suncobalt.dyndns.org:8080/browse/road_report/PL/DK86

**Comment by ppawel** [comment]
Yes, this is definitely a bug. There are more roads like this one. The problem occurs when one of the carriageways does not contain above 90% of "oneway" ways. This should be reported better.
